sendgrid/helpers/mail/personalization.py

```diff
@@ -51,11 +51,16 @@ def add_to(self, email):
                     self.add_substitution(substitution)
             else:
                 self.add_substitution(email.substitutions)
+
+        if email.dynamic_template_data:
```

Comment:
Need to check the type here to account for the case when a dictionary is passed like in `use_cases/send_multiple_emails_to_multiple_recipients.md`

Reply:
That's already account for on L206.